common/bolt11.h

```diff
@@ -14,6 +14,8 @@
 /* We only have 10 bits for the field length, meaning < 640 bytes */
 #define BOLT11_FIELD_BYTE_LIMIT ((1 << 10) * 5 / 8)
 
+#define BOLT11_F_BASE_MPP 0
```

Comment:
defined but unused. is that by design?

Reply:
A portent!  A hint, of things to come...

But I'll remove it :)